src/lang/hu-hu.json

```diff
@@ -388,7 +388,7 @@
 	"delete entries": "Biztosan töröl {count} elemet?",
 	"deleting items": "{count} elem törlése…",
 	"import project zip": "Projekt importálása zip-ből",
-	"changelog": "Változásnapló",
+	"changelog": "Változáslista",
```

Comment:
Why change list , instead of change log

Reply:
Previous translation is correct

Reply:
We have a software translation dictionary https://bkil.github.io/openscope-dict-eng-hun/#Changelog

Changelog -> is a file
Changelist -> what the file contain

https://keepachangelog.com/en/1.1.0/

What is a changelog?

A changelog is a file which contains a curated, chronologically ordered list of notable changes for each version of a project.

Reply:
Thanks for clarifying it